### DISCUSSION TITLE
A black screen flickers before the web page loads

### DISCUSSION THREAD

**Discussion by baboqoo** [cefsharp/CefSharp#4674] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

89.0.17

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net 4.5.2

### Implementation

WinForms

### Reproduction Steps

In Windows OS 7, there is no issue, but in Windows OS 10, a black screen flickers before the web page loads and then disappears.

### Expected behavior

In Windows OS 7, there is no issue, but in Windows OS 10, a black screen flickers before the web page loads and then disappears.

### Actual behavior

In Windows OS 7, there is no issue, but in Windows OS 10, a black screen flickers before the web page loads and then disappears.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
> Please only open an issue if you can reproduce the problem with version 120.1.80 or later.

Please don't raise issues for old unsupported versions.
